designs/defaults-and-model-evolution.md

```diff
@@ -277,6 +320,11 @@ The following shapes have restrictions on their default values:
 
 ### Updating default values
 
+The default value of a member MAY be changed. The default value of a root-level
+shape MUST NOT be changed since that would break any shape that refers to
+the shape, and could break other models that refer to a shape defined in a
+shared model.
+
 The default value of a member SHOULD NOT be changed. However, it MAY be
```

Comment:
It is implied that this applies only when the member has a default without the target root level shape having a default, but maybe worth calling it out.
`The default value of a member (that targets a shape without a default value) SHOULD NOT be changed.`